### PR TITLE
tc03_Cannot log into Railway with invalid password

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

File: src/test/java/tests/LoginTest.java [hunk 1]
```diff
@@ -39,4 +39,20 @@ public void tc02_UserCantLoginWithBlankUsernameTextbox() {
         LogHelper.info("Verify that user can't login with blank Username textbox");
         Assert.assertEquals(actual, errorMsg, "An error message is not display like expected result");
     }
```
Comment (tanmle) [inline src/test/java/tests/LoginTest.java:43]: code format

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/tests/LoginTest.java (hunk 1 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java:43]: fixed

File: src/test/java/tests/LoginTest.java [hunk 2]
```diff
@@ -37,6 +38,24 @@ public void tc02_UserCantLoginWithBlankUsernameTextbox() {
...
+        LogHelper.info("Enter valid email and invalid password. Click login button");
+        loginPage.login(Constant.USERNAME, "!@#$%^&***");
+
+        LogHelper.info("Get error message");
+        String actual = loginPage.getErrorMessage();
+
+        LogHelper.info("Verify that user cannot log into Railway with invalid password");
+
+        Assert.assertEquals(actual, "Invalid username or password. Please try again.",
+                "An error message is not display like expected result");
```
Comment (tanmle) [inline src/test/java/tests/LoginTest.java]: English grammar is not correct

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

**Review by MinhQuan119 — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**